src/NServiceBus.Core.Tests/Unicast/RunningEndpointInstanceTest.cs

```diff
@@ -21,8 +22,32 @@ public async Task ShouldAllowMultipleStops()
                 null);
 
             await testee.Stop();
+            
 
             Assert.That(async () => await testee.Stop(), Throws.Nothing);
+        }       
```

Comment:
There is extra whitespace added, I propose to install https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer to make it easier to detect and remove